lib/Controllers/create.js

```diff
@@ -30,6 +45,20 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      if(
+        self.include && self.include.length
```

Comment:
I don't think you need to check for any of this anymore, basically if someone specifies ```reloadOnCreate: true``` then you should just reload the instance no need to check if it has includes, or if you find the properties in context etc